server/backend/api/app/crud/crud_utils.py

```diff
@@ -90,3 +92,20 @@ def get_application_id_from_name(db, application_name):
 def raise_http_exception(status_code: str, error_msg: str):
     LOGGER.info(error_msg)
     raise HTTPException(status_code=status_code, detail=error_msg)
+
+
+def is_app_admin(
```

Comment:
Hi Ian, since you're adding the is_app_admin to the utils, do you want to remove the `is_app_admin` method from router_guard? or combine them?

Reply:
Thanks Catherine.... I was thinking to do that but then feeling hesitation afterward so created one in crud_utils.py. We can discuss, maybe it won't be issue then I originally thought....

Reply:
Hi Catherine, this is refactored as we discussed.
Also some refactoring on raise exception. Sorry for unnecessarily more files changes to make it difficult for your review. Thank!

Reply:
no worries, and Ian just wondering did we discussed if we want to keep this method under crud_utils or under the upper level utils?

Reply:
I think it is ok to leave it at crud_utils as I was afraid for some circular dependency can occur.

Reply:
sure, I just feel since we use this in both router and crud files, maybe should put it upper level? but I'm also fine to leave it, thanks!!